Fix `stylelint.utils.checkAgainstRule()` regression when both `reportNeedlessDisables` and `quiet` enabled (#7879)
stylelint/stylelint: .changeset/stale-geckos-own.md

---
"stylelint": patch
---

> Fixed: `stylelint.utils.checkAgainstRule()` to prevent false problems when both `reportNeedlessDisables` and `quiet` enabled

Reviewer: ```suggestion
Fixed: `stylelint.utils.checkAgainstRule()` regression when `reportNeedlessDisables` and `quiet` are both enabled
```
